applications/videown.md

```diff
@@ -0,0 +1,173 @@
+- Video playback
+- System login and logout
+- My Video NFT Page
+- Minting NFT page
+- Listing/UnListing NFT page
+- Buy NFT page
+- Transfer NFT page
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Hayes White
+- Justin Liu
+- Jack Chen
+
+### Contact
+
+- **Contact Name:** Hayes White
+- **Contact Email:** lostinwind@protonmail.com
+
+### Team's experience
+
+CONTRACT LAB is a Web3 team committed to providing digital asset management and Web3 technology solutions to global enterprises. Founded in February 2018, we are headquartered in Hong Kong with offices in London, Beijing, and Guangzhou. Our services include one-stop solutions, customized Web3 projects, and incubation services for Web3 startups. We use technological innovation to optimize user experience and provide secure and reliable products, striving to make entering the world of Web3 simpler for users.
+
+Our core team members come from early blockchain communities and projects, with years of technical development experience. Our solutions cover a wide range of business scenarios, including public chains, wallets, exchanges, NFTs, DeFi, and mining pools, and we are proficient in the latest technologies such as smart contracts, cross-chain, L2, ZKP, multi-signature, oracle, IOT, and more. We are well-equipped to provide the highest quality digital asset management and Web3 technology solutions to enterprises and comprehensive incubation services to Web3 startups to achieve their business goals and accelerate growth.
+
+### Team Code Repos
+
+- https://github.com/contract-lab-collections
+
+Team Members:
+
+- https://github.com/lost-inwind
+- https://github.com/JustinQP
+- https://github.com/jackchen0120
+
+## Development Status :open_book:
+
+The result of our research regarding the solutions and architecture has been described in above.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):** 2 FTE
+- **Total Costs:** 28,000 USD
+
+### Milestone — Basic functionality
+
+- **Estimated duration:** 2 months
+- **FTE:**  2
+- **Costs:** 18,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how use the product, display and explain the function of each component. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Smart contract: videown-nft | We will deliver a ink! smart contract that will realize the function of NFT market and comply with [psp-34](https://github.com/w3f/PSPs/blob/master/PSPs/psp-34.md). |
```

Comment:
Would you please provide more details on what you will build with ink!? A budget of $18k does not make sense for a single NFT contract. ink!ubator is meant to fund ink!-related projects, not APIs and infra that are unrelated.

I would like to see more interesting features on the contract side. Please elaborate on what the contract/s would do.

Reply:
Thank you for the feedback @DoubleOTheven . The contract will contain three types of functions. Not only the basic functions of NFT, but also NTF market functions and Economic model.